rosdep/base.yaml

```diff
@@ -1713,11 +1713,16 @@ libfftw3:
   ubuntu: [libfftw3-3, libfftw3-dev]
 libflann:
   arch: [flann]
-  debian: [libflann1.7]
+  debian:
+    '*': [libflann1.9]
+    jessie: [libflann1.8]
+    squeeze: [libflann1.7]
```

Comment:
I don't see `libflann1.7` in squeeze. https://packages.debian.org/squeeze/allpackages

Reply:
My bad in detecting when this package started in Debian. I see it in wheezy, though, so I'll just remove the squeeze entry.